report_qweb_decimal_place/reports/price_unit_value_format.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <template id="price_unit_value_format">
+        <t t-if="currency.apply_price_unit_format">
+            <span
+                class="text-nowrap"
+                t-esc="price_unit"
+                t-options='{"widget": "float", "precision": currency.decimal_places}'
```

Comment:
@AungKoKoLin1997 We do not want to use the field currency.decimal_places.

This https://github.com/qrtl/nrq-custom/blob/a27e08d69ae5307e61eedfb06062604baa7d990d/report_common_nrq/report/common_template.xml#L349-L369 is the minimal implementation we need.

It could nice to provide another option by adding price_decimal_places in res.currency and using the set value, however I do not see a strong need for it.

Having `apply_price_unit_format` in res.currency is good. 👍